docs/application/web/guides/security/privacy-related-permissions.md

```diff
@@ -18,6 +18,11 @@ The main features of the Privacy Privilege API include:
 
 For a list of privacy-related privileges, see [Security and API Privileges](../../tutorials/sec-privileges.md).
 
+> [!NOTE]
+>
+> Since Tizen 5.5, pop-ups by `requestPermission()` and `requestPermissions()` are launched as `group mode` with the caller apps.   
+> If the pop-up is terminated without full responses, all remained requests that aren't responded by the user will be treated as if the user selected 'Deny' for the requests. In this case, the app will get `PPM_DENY_ONCE` for those permissions and the app can request the **automatically denied** permissions again anytime it wants.  
+> If you need to request multiple privileges then use `requestPermissions()` instead of calling `requestPermission()` multiple times.
```

Comment:
@yl33 
I see there is difference in method camel case notation style. 
`requestPermission()` method starts with small **r** , while as  file docs/application/dotnet/guides/security/privacy-related-permissions.md, it starts with capital **R**. `RequestPermission()`
Are they different? should'nt we follow consistent camel case notation styling for naming methods, incase they are same? 
If my question is not valid, kindly ignore this, else, kindly do the needful.

Reply:
They are following naming rule of each language(web and dotnet). Don't need to fix.

Reply:
@yl33 thanks for clearng my doubt.